ui/widgets/layout/src/components/layout-widget.tsx

```diff
@@ -236,7 +253,7 @@ const Layout: React.FC<RootComponentProps> = props => {
               <Extension name={props.layoutConfig.pluginSlotId} uiEvents={props.uiEvents} />
             </div>
           </div>
-          <div className={tw('sticky top-0 h-screen')}>
+          <div className={tw('sticky top-0')}>
```

Comment:
same here, just replace all divs with Box and and you may safely do away with 'tw'